src/components/settings/SettingsLMSTab/LMSConfigs/BlackboardConfig.jsx

```diff
@@ -176,6 +174,7 @@ const BlackboardConfig = ({ enterpriseCustomerUuid, onClick, existingData }) =>
     // Otherwise post a new config
     } else {
       try {
+        transformedConfig.active = false;
```

Comment:
not a major thing but: do we have to do tis for each LMS implementation, or can it be done in a common place such as whatever creates the `config` before snakeCaseDict is called?  that may help centralize this state value and this code would just simply post the config ... just an idea

Reply:
one reason would be if we add a new config, implementation, we would have to remember to do this there as well, vs having in a central place, is what I thought

Reply:
It doesn't have to be done for each implementation, because the error that we're correcting is that it should only be set to false for new ones. If we attempted to centralize it, it would get reset to false even when an update was being posted.

Reply:
ah I see ok thanks for clarifying